src/goto-instrument/contracts/utils.h

```diff
@@ -11,6 +11,7 @@ Date: September 2021
 #ifndef CPROVER_GOTO_INSTRUMENT_CONTRACTS_UTILS_H
 #define CPROVER_GOTO_INSTRUMENT_CONTRACTS_UTILS_H
 
+// clang-format off
```

Comment:
Why do we need to add this only in this file? Our https://github.com/diffblue/cbmc/blob/develop/.clang-format doesn't handle includes properly already?